ProjectCoimbra.UWP/Project.Coimbra/Project.Coimbra.csproj

```diff
@@ -210,6 +224,9 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Data.Sqlite">
+      <Version>1.1.1</Version>

```

Comment:
v3.1.9 is available. See <https://www.nuget.org/packages/Microsoft.Data.Sqlite>. Can we use that instead?

Reply:
I will try to update to 3.1.9 and see how it works, I was going through some documentation online and it recommended 1.1.1 but maybe it was just an old doc.